docs/language.rst

```diff
@@ -171,8 +171,30 @@ Conditions can have the following forms
         | 'true'
         | 'false'
         | functioncall
+        | value 'is' 'defined'
         ;
 
+The ``is defined`` keyword checks if a value was assigned to an attribute or a relation of a certain entity. The following
```

Comment:
Not entirely sure, but I believe this is not limited to attributes and relations. The practical use cases might be limited, but I think you could use `is defined` on any value reference. I tried testing this using the if statement, but bumped into issue #1454.
@wouterdb can probably shed more light on this.

Reply:
I fixed it to also use attributes.